content/blog/sprint-review-2307.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Wazo Platform 23.07 Released
+date: 2023-05-17T19:55:00
```

Comment:
Maybe hardcode the date to Tuesday ?
Everything else LGTM :+1:

Reply:
Good idea. Thanks!